tests/findex.test.ts

```diff
@@ -462,3 +468,175 @@ test("upsert and search memory", async () => {
   )
   expect(results2.total()).toEqual(2)
 })
+
+test("generate non regression database", async () => {
+  const findex = await Findex()
+  const masterKey = new FindexKey(
+    Uint8Array.from(Buffer.from(FINDEX_TEST_KEY, "base64")),
+  )
+  const label = new Label(FINDEX_TEST_LABEL)
+
+  const dbFilepath = "node_modules/sqlite.db"
```

Comment:
Why not use an in memory database here?

Reply:
we need to send this database for another language verification:
- do a findex search on this database for Java, Flutter and Python
- do an upsert indexes

Reply:
Ok. Maybe name it differently or add a comment to explain that this database is the output of this function? You could also use an in memory database and at the end of the test saved it to a file with `await db.backup('database_clear.sqlite')` to avoid having the goal of this function buried in the middle?

Reply:
I added a comment at the beginning of the test.